src/NLog.MailKit/NLog.MailKit.csproj

```diff
@@ -35,7 +35,7 @@ If the mail target was already available on your platform, this package will ove
 See https://github.com/NLog/NLog.MailKit/releases
       
 </PackageReleaseNotes>
-    <AssemblyVersion>4.0.0.0</AssemblyVersion>
+    <AssemblyVersion>4.1.0.0</AssemblyVersion>
```

Comment:
See note below ;)

Reply:
That's embarrassing, sorry. Completely missed that comment and that you manage versions via CI/CD.